it also seems the `cdc/mixed-versions` test has a different failure mode. the timeout here occured after the scheduled backup cmd.
```
00:00:25 cockroach.go:847: SCHEDULED BACKUP 0
NOTICE: schedule %!q(*string=0xc003eb2340) already exists, skipping
00:00:29 versionupgrade.go:164: test status: versionUpgradeTest: starting step 12
00:00:29 mixed_version_cdc.go:162: test status: waiting for 5 resolved timestamps
00:01:42 mixed_version_cdc.go:273: 11 resolved timestamps validated, latest is 10m59.223895475s behind realtime
00:01:42 mixed_version_cdc.go:176: 1 of 5 timestamps resolved
00:03:42 mixed_version_cdc.go:273: 12 resolved timestamps validated, latest is 12m15.741057494s behind realtime
00:03:42 mixed_version_cdc.go:176: 2 of 5 timestamps resolved
I230224 00:05:17.895588 1592 github.com/Shopify/sarama/sarama.go:122  [T1,kafka-producer] 45  client/metadata fetching metadata for all topics from broker 35.245.39.176:9092
00:05:26 test_impl.go:344: test failure #1: full stack retained in failure_1.log: (test_runner.go:989).runTest: test timed out (30m0s)
I230224 00:05:41.138285 1514 github.com/Shopify/sarama/sarama.go:122  [T1,kafka-producer] 46  client/metadata fetching metadata for all topics from broker 35.245.39.176:9092
I230224 00:05:55.405181 1784 github.com/Shopify/sarama/consumer.go:911  [T1,kafka-producer] 47  consumer/broker/0 accumulated 1 new subscriptions
I230224 00:05:55.405331 1785 github.com/Shopify/sarama/consumer.go:951  [T1,kafka-producer] 48  consumer/broker/0 added subscription to bank/0
I230224 00:05:55.405367 1785 github.com/Shopify/sarama/consumer.go:957  [T1,kafka-producer] 49  consumer/broker/0 closed dead subscription to bank/0
I230224 00:05:55.405411 1691 github.com/Shopify/sarama/sarama.go:125  [T1,kafka-producer] 50  Closing Client
I230224 00:05:55.405497 27629 github.com/Shopify/sarama/sarama.go:122  [T1,kafka-producer] 51  Closed connection to broker 35.245.39.176:9092
I230224 00:05:55.405559 27628 github.com/Shopify/sarama/sarama.go:122  [T1,kafka-producer] 52  Closed connection to broker 35.245.39.176:9092
--- FAIL: cdc/mixed-versions (1829.35s)
test artifacts and logs in: artifacts/cdc/mixed-versions/run_1
(test_runner.go:989).runTest: test timed out (30m0s)
(mixed_version_cdc.go:270).1: read tcp 10.142.0.85:33362 -> 34.86.66.147:26257: read: connection reset by peer
00:05:56 test_runner.go:726: [w0] test failed: cdc/mixed-versions (run 1)
```

--- a/pkg/cmd/roachtest/tests/multitenant_upgrade.go
+++ b/pkg/cmd/roachtest/tests/multitenant_upgrade.go
@@ -130,7 +130,9 @@ func runMultiTenantUpgrade(ctx context.Context, t test.Test, c cluster.Cluster, 
 	t.Status("upgrading system tenant binary")
 	c.Stop(ctx, t.L(), option.DefaultStopOpts(), kvNodes)
 	settings.Binary = currentBinary
-	c.Start(ctx, t.L(), option.DefaultStartOpts(), settings, kvNodes)
+	// TODO (msbutler): investigate why the scheduled backup command fails due to a `Is the Server
+	// running?` error.
+	c.Start(ctx, t.L(), option.DefaultStartOptsNoBackups(), settings, kvNodes)
 	time.Sleep(time.Second)
 
 	t.Status("checking the pre-upgrade sql server still works after the system tenant binary upgrade")

--- a/pkg/roachprod/install/cockroach.go
+++ b/pkg/roachprod/install/cockroach.go
@@ -826,15 +826,26 @@ func (c *SyncedCluster) createFixedBackupSchedule(
 	createScheduleCmd := fmt.Sprintf(`CREATE SCHEDULE IF NOT EXISTS test_only_backup FOR BACKUP INTO '%s' %s`,
 		collectionPath, scheduleArgs)
 
-	node := Node(1)
+	node := c.Nodes[0]
 	binary := cockroachNodeBinary(c, node)
 	url := c.NodeURL("localhost", c.NodePort(node), "" /* tenantName */)
 	fullCmd := fmt.Sprintf(`COCKROACH_CONNECT_TIMEOUT=0 %s sql --url %s -e %q`,
 		binary, url, createScheduleCmd)
-	// Instead of using `c.ExecSQL()`, use the more flexible c.Run(), which allows us to
+	// Instead of using `c.ExecSQL()`, use the more flexible c.newSession(), which allows us to
 	// 1) prefix the schedule backup cmd with COCKROACH_CONNECT_TIMEOUT=0.
-	// 2) run the command against the first node on the cluster.
-	return c.Run(ctx, l, l.Stdout, l.Stderr, Nodes{node}, "init scheduled backup", fullCmd)
+	// 2) run the command against the first node in the cluster target.
+	sess := c.newSession(l, node, fullCmd, withDebugName("init-backup-schedule"))
+	defer sess.Close()
+
+	out, err := sess.CombinedOutput(ctx)
+	if err != nil {
+		return errors.Wrapf(err, "~ %s\n%s", fullCmd, out)
+	}
+
+	if out := strings.TrimSpace(string(out)); out != "" {
+		l.Printf(out)
+	}
+	return nil
 }
 
 // getEnvVars returns all COCKROACH_* environment variables, in the form

